lib/WebDriverCheckbox.php

```diff
@@ -0,0 +1,296 @@
+<?php
+// Copyright 2004-present Facebook. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Facebook\WebDriver;
+
+use Facebook\WebDriver\Exception\NoSuchElementException;
+use Facebook\WebDriver\Exception\UnexpectedTagNameException;
+use Facebook\WebDriver\Exception\UnsupportedOperationException;
+use Facebook\WebDriver\Exception\WebDriverException;
+use Facebook\WebDriver\Support\XPathEscaper;
+
+/**
+ * Provides helper methods for checkboxes and radio buttons.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class WebDriverCheckbox implements WebDriverSelectInterface
```

Comment:
I suggest splitting the class in two (Checkbox and Radio), because:
- the semantic is confusing (`$radiobutton = new WebDriverCheckbox($radibuttonElement)`)
- there are lot of `$this->isMultiple()` conditionals, which separates logic for checkboxes and radiobuttons

Maybe some common abstract class could be introduced to DRY the logics. What do you think?